src/lilURL.php

```diff
@@ -484,6 +487,12 @@ public function resetRedirectCount($id) {
         $statement = $this->executeQuery($sql, [
             ':urlID' => $id,
         ]);
+
+        $sql = 'UPDATE '.$this->getUrlTable().' set lastRedirectDate = NULL where urlID = :urlID';
```

Comment:
Don't need to change, but instead of adding an additional update statement and call to the DB you should have just updated the existing update to also set `lastRedirectDate = NULL`

`$sql = 'UPDATE '.$this->getUrlTable().' set redirects = 0,  lastRedirectDate = NULL where urlID = :urlID';`